core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.component.css

```diff
@@ -122,10 +122,14 @@
   padding-top: 27px;
 }
 
+.oppia-top-navigation-bar.navbar-header {
+  float: left !important;
```

Comment:
Why is `!important` needed?

Reply:
had just added float-left css class's css here after removing the class from html code, removed important, it was't required. Thanks!